core/HoistModel.js

```diff
@@ -62,6 +62,7 @@ export class HoistModel extends HoistBase {
     @observable
     _componentProps = {};
     _modelLookup = null;
+    _created = Date.now();
```

Comment:
Seems like a useful / harmless-enough property to install generally. We could prefix with `xh` if that seemed safer, but we weren't doing that for the other `_` props we set here.

Reply:
agree